src/gp-model/helpers/get_data.py

```diff
@@ -120,15 +132,14 @@ def get_pm_data(device_id, owner, verbose=True, frequency = 'raw', start_time='2
     
     """
 
-    lat, lon, name = get_device_details(device_id, owner)
-    url = EVENTS_URL #AirQo Platform Get Events endpoint
-    result = [] #array to store all data downloaded for a device
-    measurements_length= 1000 #standard response length from api
-    count = 0 #iteration number
+    url = EVENTS_URI
```

Comment:
As we continue utilise the Events URI, please note that you will not longer need to specify recent=no in the near future when querying data. It will be the default going forward. In addition to making hourly values as the default. So future queries for raw values would have to specify the frequency query parameter to escape using hourly measurements when you needed to use raw ones.  I hope this does not bring any complications. Related PR is [this one.](https://github.com/airqo-platform/AirQo-api/pull/604)